docs/guide/architecture-styles/event-driven-content.md

```diff
@@ -41,6 +41,7 @@ In the logical diagram above, each type of consumer is shown as a single box. In
 
 - Guaranteed delivery. In some systems, especially in IoT scenarios, it's crucial to guarantee that events are delivered.
 - Processing events in order or exactly once. Each consumer type typically runs in multiple instances, for resiliency and scalability. This can create a challenge if the events must be processed in order (within a consumer type), or if the processing logic isn't idempotent.
+- Coordinating messages across services. Business processes often involve multiple services publishing and subscribing to messages to achieve a consistent outcome across a whole workload. [Workflow patterns](https://docs.particular.net/architecture/workflows/) can be used to reliably manage message flows across various services.
```

Comment:
This link doesn't work. [docs.particular.net/architecture/workflows](https://docs.particular.net/architecture/workflows/) results in a 404.  The `/` breaks it.

```suggestion
- Coordinating messages across services. Business processes often involve multiple services publishing and subscribing to messages to achieve a consistent outcome across a whole workload. [Workflow patterns](https://docs.particular.net/architecture/workflows) can be used to reliably manage message flows across various services.
```

Reply:
good catch, thanks ❤️